packages/hash/frontend/src/components/layout/PageHeader/NotificationsDropdown.tsx

```diff
@@ -0,0 +1,159 @@
+import { useState, useRef } from "react";
+import { faBell } from "@fortawesome/free-solid-svg-icons";
+import {
+  Box,
+  IconButton,
+  ListItemButton,
+  Typography,
+  useTheme,
+} from "@mui/material";
+
+import { FontAwesomeSvgIcon } from "../../icons";
+import { Popover } from "../../Popover";
+import { Link } from "../../Link";
+
+export const NotificationsDropdown: React.FC = () => {
+  const theme = useTheme();
+
+  const buttonRef = useRef(null);
+
+  const [open, setOpen] = useState(false);
+  const [notificationsLength] = useState(0);
+
+  const hasNotifications = !!notificationsLength;
+
+  const id = open ? "actions-popover" : undefined;
+
+  return (
+    <Box>
+      <IconButton
+        sx={{
+          mr: {
+            xs: 1,
+            md: 1.5,
+          },
+          fontSize: theme.spacing(2),
+          width: hasNotifications ? "auto" : "32px",
+          px: hasNotifications ? 1.5 : "unset",
+          height: "32px",
+          borderRadius: hasNotifications ? 4 : "100%",
+          color:
+            hasNotifications || open
+              ? theme.palette.common.white
+              : theme.palette.gray[40],
+          backgroundColor:
+            hasNotifications || open
+              ? theme.palette.blue["70"]
+              : theme.palette.gray[20],
+        }}
+        ref={buttonRef}
+        onClick={() => setOpen(!open)}
+      >
+        <FontAwesomeSvgIcon icon={faBell} />
+        {hasNotifications && (
+          <Box component="span" ml={1}>
```

Comment:
Please use the `Typography` component here, and note the designs have a greater font weight:
<img width="78" alt="image" src="https://user-images.githubusercontent.com/42802102/156575519-01220461-35a9-4c43-89c2-f12ce8cac2f7.png">
<img width="130" alt="image" src="https://user-images.githubusercontent.com/42802102/156575588-e24cdf79-0ed0-462e-b244-a191b408051c.png">